File: src/test/samples/sample.pas
//#region FirstRegion
program HelloWorld;
begin
  writeln('Hello, World!');
end.
// #endregion

{  #region    SecondRegion  }
procedure MyProcedure;
begin
  (*    #region    InnerRegion    *)
  writeln('Inside region');
  (*  #endregion   ends InnerRegion  *)

  {#region}
  writeln('Another unnamed region');
  {#endregion}
end;
{   #endregion   }

File: src/test/samples/sample.r
#region FirstRegion
x <- 42
#endregion

# region SecondRegion
my_function <- function() {
  #region      InnerRegion
  print("Hello")
  #endregion      ends InnerRegion

 #   region
 print("Unnamed region")
    #endregion
}
# endregion
